README.md

```diff
@@ -231,12 +231,12 @@ Examples to start the fictitious file named `foo_measurement.py`:
 
 ### Create Executable for Python Scripts
 
-To create an executable from a measurement, measurement authors can use the [pyinstaller](https://www.pyinstaller.org/) tooling. During the executable creation, the user can also embed the User Interface file using the `--add-data "<path_of_the_UI_File>;."`.
+To create an executable from a measurement, measurement authors can use the [pyinstaller](https://www.pyinstaller.org/) tooling. If you are using a virtual environment, ensure that pyinstaller is installed to your venv. During the executable creation, the user can also embed the User Interface file using the `--add-data "<path_of_the_UI_File>;."`. The `.serviceconfig` file will need to be included using the `--add-data` option as well.
```

Comment:
Remove random capitalization.

```suggestion
To create an executable from a measurement, measurement authors can use the [pyinstaller](https://www.pyinstaller.org/) tooling. If you are using a virtual environment, ensure that pyinstaller is installed to your venv. During the executable creation, the user can also embed the user interface file using the `--add-data "<path_of_the_UI_file>;."`. The `.serviceconfig` file will need to be included using the `--add-data` option as well.
```

Reply:
These are no longer mentioned as the update to the code means it is no longer recommended to use `--add-data` for these files.